GPflow/priors.py

```diff
@@ -88,6 +104,10 @@ def __init__(self, mu, sigma):
     def logp(self, x):
         return tf.reduce_sum(densities.laplace(self.mu, self.sigma, x))
 
+    def sample(self, shape=()):
+        return np.array(np.random.laplace(self.mu, self.sigma,
+                                          size=shape if shape else None))
```

Comment:
Why wrap this in np.array? wouldn't it be neater to have the shape default to (1,) ?